app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -251,6 +253,7 @@ class ExplorationActivityTest {
 
   private fun getApplicationDependencies(
```

Comment:
it seems this function is not used anywhere, and could probably be safely removed.

Reply:
Done.